sdlib/d/gc/extent.d

```diff
@@ -357,6 +359,79 @@ public:
 		return readPackedFreeSpace(freeSpacePtr(index));
 	}
 
+	size_t getReservedCapacity(uint index) {
+		assert(isSlab(), "getReservedCapacity accessed on non slab!");
+		assert(sizeClassSupportsMetadata(sizeClass),
+		       "size class not supports slab metadata!");
+		assert(index < slotCount, "index is out of range!");
+
+		if (!hasFinalizer(index)) {
+			return 0;
+		}
+
+		return PointerSize;
+	}
+
+	enum FinalizerBit = nativeToBigEndian!size_t(size_t(0x2));
```

Comment:
You always use a constructor here because somehow VRP doesn't kick in when calling the function, right?